NOCList-Obj-C/NOCList-Obj-C/LSIAgent.h

```diff
@@ -0,0 +1,35 @@
+//
+//  LSIAgent.h
+//  NOCList-Obj-C
+//
+//  Created by Lambda_School_Loaner_268 on 5/11/20.
+//  Copyright © 2020 Lambda. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface LSIAgent : NSObject
+
+
+// Properties
+@property NSString *coverName;
+
+@property NSString *realName;
+
+@property NSNumber *accessLevel;
+
+@property NSNumber *compromised;
```

Comment:
I think you could leave out the newlines between properties, but that's a personal preference thing.